helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -113,25 +113,17 @@ internal class CreateCommand : CliktCommand(
     }
 
     private fun createPackageConfiguration(provenance: Provenance): PackageConfiguration =
-        when (provenance) {
-            is ArtifactProvenance -> {
-                PackageConfiguration(
-                    id = packageId,
-                    sourceArtifactUrl = provenance.sourceArtifact.url
+        PackageConfiguration(
+            id = packageId,
+            sourceArtifactUrl = provenance.let { it as? ArtifactProvenance }?.let {
+                it.sourceArtifact.url
+            },
+            vcs = provenance.let { it as? RepositoryProvenance }?.let {
```

Comment:
Simplify to: `(provenance as? RepositoryProvenance)?.let {`

Reply:
It seems while you reviewed I merged already. I've addressed this with https://github.com/oss-review-toolkit/ort/pull/5687.